floodviz/map_utils.py

```diff
@@ -113,3 +50,48 @@ def projection_info(code, url):
     if req.status_code != 200:
         return None
     return req.text
+
+
+def site_dict(ids, url_prefix):
+    """
+    Retrieves data on a list of sites and returns it in a list of dicts
+    :param ids: list of site IDs
+    :param url_prefix: NWIS site url prefix as a string
+    :return: list of dictionaries describing the sites
+    note that huc_cd, along with some other fields, will not be included
+    """
+    if not ids:
+        print('No site IDs provided')
+        return []
+
+    # generate the string of site ids for the url
+    id_input_string = ",".join(ids)
+    url = url_prefix + "site/?format=mapper&sites=" + id_input_string + "&siteStatus=all"
```

Comment:
I'm going to push back on this. The mapper format is being used for a specific application in the USGS. I'd like to stick to the rdb format as in it well documented and includes complete information.